src/Talis/Persona/Client/InvalidValidationException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace Talis\Persona\Client;
+
+/**
+ * Validating a token failed.
+ */
+class InvalidValidationException extends \Exception
```

Comment:
How about `TokenValidationException`?

Reply:
is this only about tokens?  The validation might fail because the token is wrong, or because the certificate is invalid.

I'd be tempted to make this `PersonaValidationException` on that basis.